astro/src/components/get-started/GitPodStart.astro

```diff
@@ -0,0 +1,66 @@
+---
+import Aside from 'src/components/Aside.astro';
+import Code from 'astro/components/Code.astro';
+import { RemoteValue } from '@fusionauth/astro-components';
+
+export interface Props {
+  /**
+   * Full URI to extract values from kickstart.json
+   * e.g. https://raw.githubusercontent.com/FusionAuth/fusionauth-quickstart-ruby-on-rails-web/main/kickstart/kickstart.json
+   */
+  kickstartUri: string;
+
+  /**
+   * Flag that indicates whether this in API quickstart or not (default)
+   */
+  api?: boolean;
+}
+
+const { kickstartUri, api = false } = Astro.props as Props;
+---
+<h2> Install/Run in GitPod</h2>
+<dd>
+<a href="https://gitpod.io/#https://github.com/synedra/fusionauth-example-express-start-here" alt="Open in GitPod"><img src="https://gitpod.io/button/open-in-gitpod.svg" /></a>
```

Comment:
This should open in a new tab, right? Or is the idea to take them entirely away from the FusionAuth site?

Reply:
Added target="_blank" to get it in a separate tab.  GitPod is a separate site, but we can try to soften this experience for people.